extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/music/GoogleMusicHttpApi.java

```diff
@@ -213,23 +220,19 @@ private <T> T makePutRequest(
     // Wait for write permit before making request
     writeRateLimiter.acquire();
 
-    HttpRequestFactory requestFactory = httpTransport.createRequestFactory();
-    HttpRequest putRequest =
-        requestFactory.buildPutRequest(
-            new GenericUrl(baseUrl + "?" + generateParamsString(parameters)), httpContent);
-    putRequest.setReadTimeout(2 * 60000); // 2 minutes read timeout
+    HttpRequestFactory requestFactory = httpTransport.createRequestFactory(credential);
+    HttpRequest patchRequest = buildPatchRequest(requestFactory, baseUrl,
+        generateParamsString(parameters), httpContent);
+    patchRequest.setReadTimeout(2 * 60000); // 2 minutes read timeout
```

Comment:
is this timeout purposely not on the retry version? I wonder if we should just comment that so it doesn't look like a mistake? (eg: "timeout only here and not in our retry because ...')

Reply:
Remove the read timeout from Patch request since createPlaylist will not have too much data for server to read, which is different from batchCreatePlaylistItems.